graphql/schema.graphql

```diff
@@ -71,4 +71,22 @@ type Query {
     """
     sponsoredCount: Int
   ): [SponsoredProduct] @cacheControl(scope: PUBLIC, maxAge: SHORT)
+  sponsoredBanners(
+    """
+    Identifier for users, logged in or not. Used for A/B tests.
+    """
+    anonymousId: String
+    """
+    Placement of the banner. Used for analytics purposes.
+    """
+    placement: String
```

Comment:
If this placement have fixed values like HOME, PRODCT_PAGE, etc, I would recommend it to be `Enum` instead of a `String`.

Reply:
Yes, it's the case. I'll adjust it.